core/ui/prime/components/src/lib/tree/tree.component.ts

```diff
@@ -156,4 +157,19 @@ export class TreeComponent extends InputBaseComponent {
 
 		this.valueChanged();
 	}
-}
+
+	public override setDisabledState(isDisabled: boolean): void {
+		this.disabled = isDisabled;
+		for (const node of this.hierarchicalNodes) {
+			this.setNodeSelectableRecursive(node, !isDisabled);
+		}
+	}
+
+	private setNodeSelectableRecursive(node: TreeNode<ITreeNode>, isSelectable: boolean): void {
+		node.selectable = isSelectable;
+		if (!node.children) return;
```

Comment:
Use {} even for single line statements to prevent future mistakes when adding extra statements

Reply:
Fixed